ui-v2/app/components/policy-form/index.hbs

```diff
@@ -12,9 +12,9 @@
     {{! this should use radio-group }}
     <div role="radiogroup" class={{if item.error.Type ' has-error'}}>
         {{#each templates as |template|}}
-            <label>
+            <label data-test-radiobutton={{concat 'template_' template.template}}>
```

Comment:
use `for` for accessibility
```suggestion
            <label data-test-radiobutton={{concat 'template_' template.template}} for={{concat name '[template]'}}>
```

Reply:
Pretty sure `for` attributes only refer to `id`s?

> The first element in the document with an id matching the value of the for attribute is the labeled control for this label element, if it is a labelable element.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for

We'd have to then also add an `id` to the input, and then make sure the id was unique, this specific component gets nested and duplicated in various places so that could get problematic.

Totally could be looked at some point though, just this PR was supposed to be only moving files around 😬 !

Generally most of the labels we use are implicitly labelled:

> In this case, the label element is used as a container for both the form control and the label text, so that the two are associated implicitly. Generally, explicit labels are better supported by assistive technology.

https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly

I'd actually love to know what the level of support is for implicit labels across different assistive technologies though.